### ISSUE TITLE
Create fuzz testing crate for key presses

### ISSUE THREAD

**Issue by friendlymatthew** [gaucho-labs/wasm-keys#1]
@mondeja what do you think?

The goal is to split `leptos-hotkeys` and use this library for types. This crate implements the `From` trait for `web_sys::Keyboard_Event`. It also does the `Hotkey` creation here. 

Will add you as a maintainer to this as well

**Comment by mondeja** [comment]
Yes, that's a great idea :+1: I'll add CI here when you split leptos-hotkeys.
